react-events/my-app/src/App.js

```diff
@@ -0,0 +1,34 @@
+// import logo from './logo.svg';
+import './App.css';
+import CustomButton from './CustomButton.js'
+
+function App() {
+
+  function handleCustomClick(text) {
+    alert(text)
+  }
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <CustomButton
+          color="red"
+          text="Good Morning"
+          onCustomClick={() => handleCustomClick('Clicked red Good Morning')}
```

Comment:
You shouldn't be passing a callback function. Inside your `handleCustomClick` function, that's where you should have the logic for `Clicked color`. We want to take advantage of the `text` & `color` props that get passed down, and not hard code them here. You should just be passing a reference to the function.

`onCustomClick={handleCustomClick}`